client/packages/system/src/Stock/Components/Ledger/useLedgerColumns.ts

```diff
@@ -19,6 +19,8 @@ export enum ColumnKey {
   'Number' = 'number',
 }
 
+const REVERSED_PRESCRIPTION = 'reverse_prescription';
```

Comment:
Not stoked to define this in two places... but haven't seen us define types for constants in graphql layer anywhere else?